packages/lesswrong/lib/utils/timeUtil.ts

```diff
@@ -11,10 +11,10 @@ export type SSRMetadata = {
   timezone: string;
 }
 
-export type TimeOverride = Pick<SSRMetadata, "cacheFriendly" | "timezone"> & {
-  currentTime: Date;
+export type EnvironmentOverride = Partial<SSRMetadata> & {
+  matchSSR: boolean;
```

Comment:
There was already one thing in here which was not strictly time-related (`cacheFriendly`), and now we're using it to decide whether to render things that are no-ssr-ed, so I renamed it to something more generic